servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/QueueDescriptionSerializer.java

```diff
@@ -269,6 +266,8 @@ private static QueueDescription parseFromEntry(Node xEntry) {
                                     case "AuthorizationRules":
                                         qd.authorizationRules = AuthorizationRuleSerializer.parseAuthRules(element);
                                         break;
+                                    default:
```

Comment:
[SpotBugs-P2] 
This method contains a switch statement where default case is missing. Usually you need to provide a default case.